databricks_cli/sdk/service.py

```diff
@@ -794,3 +794,104 @@ def get_instance_pool(self, instance_pool_id=None, headers=None):
     def list_instance_pools(self, headers=None):
         _data = {}
         return self.client.perform_query('GET', '/instance-pools/list', data=_data, headers=headers)
+
+
+class AccountsService(object):
```

Comment:
Was this change autogenerated?

Reply:
No @andrewmchen, I implemented it. Sorry, should it have been auto-generated somehow?

Reply:
As per @andrewmchen - this needs to be inlined in the proto as this is auto-generated.